### PR TITLE
Add config file to provide notification URL

### PR DESCRIPTION
**Pull request by zeusongit**
This PR adds an environment file that will help us configure notification URL.

### CODE REVIEW

**Review by QilongTang — COMMENTED** [review]
This for sure looks like a step up. Do you think it will make Dynamo to consume the bundle differently?

**Comment by QilongTang** [comment]
Discussed with @zeusongit that we will use `dotenv-webpack` to introduce different endpoint for dev, staging. prod env so this repo is prod ready

**Comment by QilongTang** [comment]
@zeusongit Please address merge conflicts and update readme for this change. Once you have it tested, LGTM

File: package.json [hunk 1]
```diff
@@ -29,15 +29,16 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@filipeop/notifications-panel": "^0.0.2",
     "@babel/core": "^7.18.6",
     "@babel/preset-env": "^7.18.6",
     "@babel/preset-react": "^7.18.6",
+    "@filipeop/notifications-panel": "^0.0.2",
```
Comment (zeusongit) [inline package.json:35]: Are we still using this package?

Comment (QilongTang) [reply package.json:35]: Yes, we ran into some difficulty having @filipeotero to publish under dynamods scope..

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "lint:fix": "eslint src/ tests/ --fix",
     "test": "npm run lint:fix && jest",
     "start": "webpack serve",
-    "build": "webpack --config webpack.config.js --mode=development",
+    "build": "webpack --config webpack.config.js --env dev --mode=development",
```
Comment (QilongTang) [inline package.json:21]: please update version to `0.0.11`

**Review by QilongTang — APPROVED** [review]

**Pull request merged**